communication/youtube/youtube-guidelines.md

```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
+<conduct@kubernetes.io>.
 
-- Playlists should be organized chronologically for ease of use. This can be
-  done by updating the default ordering of the Playlist:
-  - From within the Playlist settings, click on the **Basic** Tab.
-  - From the **Ordering** dropdown select "Date added (newest)".
-  - Save the changes and the order should automatically be updated.
+### Trimming and Editing Recordings
 
-- Please do not remove any already-published content from the playlists without
-  checking with the YouTube admins.
+YouTube admins are asked to help [trim][] and [edit][] recordings that come into
+the video queue.
 
-- For any small issues that arise, for example improper naming or ordering, you
-  may be asked by the YouTube admins to attempt to resolve the issue yourself.
+#### Examples:
 
-- Any egregious or habitual violations (3 or more per quarter) of the above
-  rules will result in suspension of collaboration privileges for the particular
-  individual or for the entire playlist if the individual can’t be identified.
-  - If an individual is suspended, the playlist link will be remade and the new
-    link will be shared with the non-offending individuals.
-  - If playlist collaboration is suspended, the uploading and management of
-    the playlist will be handled by the YouTube admins. Uploading the
-	problematic group's playlist will not be considered a priority, and delays
-	in uploading should be expected.
+Certain events such as the Contributor Summits are not uploaded directly to
+YouTube and require editing.
 
+A Zoom recording may have significant dead-space leading the meeting itself and
+also at the end as we end the stream.
 
-## Admin Responsibilities  
+A SIG Meeting needs to be edited to make sure it's clear (ie "Kubernetes Special
+Interest Group ContribEx 20220131 Meeting").
 
-Purpose: Help maintain a robust YouTube channel that is valuable to contributors
-and upholds our transparency goals as laid out by our governance docs.  
+Make sure to save a copy of the video first before making adjustments as this
+can delete the original if not careful. When in doubt, ask.
 
-### Moderator Expectations and Guidelines
+### Automation
 
-Moderators should adhere to the general Kubernetes project
-[moderation guidelines].
+**Note:** There is always room for improvement! As such, the community is open
+to trying various integration features or other productivity tools that might
+improve the job of admins and help make things more streamlined. Please, reach
+out if you can help or have any ideas.
 
-Moderation responsibilities for YouTube admins is minimal and is centered around
-checking and removing any potential comments that would violate the [Code of
-Conduct][coc]. Any potential violations should sent to <conduct@kubernetes.io>.
+#### Splain.io
 
-### Trimming and Editing Recordings
+One feature used by several SIGs is splain.io. This tool creates a pipeline
```

Comment:
We may need to remove this because splain is very buggy and more often than not does not work

Reply:
I'd leave as is till we get off splain =/ YT admins may still need the instructions to troubleshoot (I've had to re-setup splain for several groups to "restart" their pipeline)